### PR TITLE
Cleanup according to Xenit's code style conventions and best practices.

### PR DESCRIPTION
**Pull request by anghelutar**

### CODE REVIEW

File: alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ private void registerCoreStats() {
                 Thread.currentThread().sleep(10_000);
                 trackerRegistry = coreAdminHandler.getTrackerRegistry();
             } catch (InterruptedException e) {
-                e.printStackTrace();
+                logger.error(e.getMessage());
```
Comment (kerkhofsd) [inline alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java:34]: use `logger.error("Short text describing what failed", e)`

**Review by kerkhofsd — CHANGES_REQUESTED** [review]
Great, thanks for cleaning this up!!

One small remark about logging exceptions.

File: alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java (hunk 1 above)
Comment (anghelutar) [reply alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java:34]: Hmm, can we leave this for latter? 
There are other parts of the code which need some cleaning and I planned to use the released artifact today.

Comment (anghelutar) [reply alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java:34]: Ignore, I'll do it.

Comment (kerkhofsd) [reply alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java:34]: You can replace all occurrences with:  `logger.error(e.getMessage(), e)`

Comment (kerkhofsd) [reply alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java:34]: That should do it.

Comment (anghelutar) [reply alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/monitoring/binder/SolrCoreStatsMetrics.java]: No problem, I added explicit messages.

**Review by kerkhofsd — APPROVED** [review]

**Pull request merged**